runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -355,8 +374,67 @@ ir::OperandIndex BaseLoader<LoaderDomain, SpecificLoader>::loadOperand(const Ten
     if (details != nullptr)
       throw std::runtime_error("Custom Quantization is not supported");
   }
+  // Sparsity
+  auto src_sparsity = tensor->sparsity();
+  int w0_dense_size = 0;
+  std::vector<uint16_t> w1_segments;
```

Comment:
This vector is created every time even if operand is not sparse. It might produce some overhead.

Reply:
@periannath I guess it would be almost zero overhead for creating empty vector. Do you have any suggestion?

Reply:
@glistening empty `std::vector` takes 24 bytes on x64 linux.

```bash
std::cout << sizeof(std::vector<int>) << std::endl;
```

I agree with you. This overhead seems negligible.

>  Do you have any suggestion?

Maybe read sparsity after creating `type_info`?

Reply:
@periannath I've updated. Looks much better. Thanks.